change/office-ui-fabric-react-2019-09-20-15-03-54-typo-combobox-docs.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
I believe this would be a "patch" rather than a "minor" change

Reply:
Updated it in 568f29090a8c23ce06748686a3d9701b05fd7c26.